web/cypress/integration/route/search-route.spec.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-loop-func */
+/* eslint-disable @typescript-eslint/no-invalid-this */
```

Comment:
Hi, do we need this?

Reply:
yes otherwise it will end up with so much of warnings.

Reply:
Do we have the same comments in other files? If no, we may need @guoqqqi to have a check.

Reply:
okay @guoqqqi could you please give a check to this

Reply:
I don't find the same comment elsewhere and I don't quite understand what this comment does, what type of warning would appear, please?